backend/src/main/resources/application.yml

```diff
@@ -4,17 +4,35 @@ micronaut:
   server:
     port: ${devmarkt.server.port:8080}
     host: ${devmarkt.server.host:`localhost`}
+  security:
+    token:
+      jwt:
+        signatures:
+          secret:
+            generator:
+              secret: ${devmarkt.auth.jwtsecret}
+              jws-algorithm: HS256
+        generator:
+          access-token:
+            expiration: 43200
```

Comment:
If generated manually (like in DevelopmentStartup) we need to handle expiration ourselves with the `exp` claim. This setting does not handle it for us